samples/react-pnp-js-sample/src/webparts/pnPjsExample/components/PnPjsExample.tsx

```diff
@@ -9,8 +9,9 @@ import { Caching } from "@pnp/queryable";
 import { getSP } from "../pnpjsConfig";
 import { SPFI, spfi } from "@pnp/sp";
 import { Logger, LogLevel } from "@pnp/logging";
-import { IItemUpdateResult } from "@pnp/sp/items";
-import { Label, PrimaryButton } from '@microsoft/office-ui-fabric-react-bundle';
+import "@pnp/sp/items";
+import { IItemUpdateResultData } from "@pnp/sp/items/types"; // Corrected import
+import { Label, Button } from "@fluentui/react-components";
```

Comment:
but we did not add the `@fluentui` to that `package.json` so that it most probably will present an error when building

Reply:
@Adam-it We should switch button away @microsoft and use @fluentui instead.  The solution failed to build using @microsoft/office-ui-fabric-react.  There is a [verbiage](https://learn.microsoft.com/en-us/sharepoint/dev/spfx/office-ui-fabric-integration#office-ui-fabric-react) about incompatibility of SPfx with Fabric React version 2.x or older. (Fabric React versions 2.x or older are not supported in SharePoint Framework.)